frontend/src/pages/Homepage.jsx

```
@@ -3,22 +3,28 @@ import Navbar from '../Components/Navbar'
+            Result<br />
+            with us!
+          </h1>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do 
              eiusmod tempor incididunt ut labom et dolore magna aliqua. Ut 
              enim ad minim veniam</p>
-          <button className="explore-btn">
+          <button 
+            className="explore-btn"
+            onClick={() => navigate('/contributions')}
+          >
             <span className="dot"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)